api/controllers/DashboardController.js

```diff
@@ -16,275 +19,400 @@ var dashboardController = {
-                                        results[i].app_status = appRef[results[i].unique_app_id];
-                                        results[i].tracking_ref = trackRef[results[i].unique_app_id];
-                                    }
-                                }
-                            }
+                    // For each element in the database results array, add the application reference status
+                    // if one exists.
+
+                    for (let result of results) {
+                        const appStatus = appRef[result.unique_app_id];
+                        result.app_status =
+                            dashboardController._userFriendlyStatuses(
+                                appStatus,
+                                result.applicationtype
+                            );
+                        result.tracking_ref = trackRef[result.unique_app_id];
+                    }
+                }
+
+                const pageAttributes = {
+                    message: req.flash('info'),
+                    users_applications: results,
+                    moment,
+                    offset,
+                    sortOrder,
+                    paginationMessage,
+                    currentPage,
+                    totalPages,
+                    searchCriteria,
+                    user_data: userData,
+                    application_total: totalApplications,
+                };
+
+                return dashboardController._redirectToPage(
+                    pageAttributes,
+                    req,
+                    res
+                );
+            }
+        );
+    },
 
-                            var moment = require('moment');
-                            var pageUpperLimit = (offset + pageSize);
-                            if (pageUpperLimit > resultCount) {
-                                pageUpperLimit = resultCount;
-                            }
-                            var totalPages = resultCount%pageSize===0? resultCount / pageSize: Math.floor(resultCount / pageSize) + 1;
-                            var paginationMessage;
-                            if (resultCount === 0) {
-                                paginationMessage = 'No applications found';
-                            }
-                            else {
-                                paginationMessage = 'Showing ' + (offset + 1) + ' &ndash; ' + pageUpperLimit + ' of ' + resultCount + ' applications submitted in the last 60 days';
-                            }
+    _userFriendlyStatuses(casebookStatus, applicationtype) {
+        const eAppStatuses = {
+            Done: {
+                text: 'Completed',
+                colorClass: '', // dark blue
+            },
+            default: {
+                text: 'In progress',
+                colorClass: 'govuk-tag--blue',
+            },
+        };
+
+        const standardStatuses = {
+            Despatched: {
```

Comment:
'Dispatched' is probably the more common way to spell it. Unless this has been passed down from CASEBOOK?

Reply:
Yeah this value is passed down from Casebook. I think that's just they way the prefer to spell it 🤷 
<img width="1103" alt="Screenshot 2021-10-04 at 18 02 16" src="https://user-images.githubusercontent.com/1377253/135893417-8707d000-510b-462e-9321-82a9e34c1400.png">